composer.json

```diff
@@ -11,31 +11,34 @@
     "homepage": "https://github.com/sulu/SuluSyliusConsumerBundle",
     "license": "MIT",
     "require": {
-        "php": "^7.1.3",
+        "php": "^7.2",
+        "friendsofsymfony/rest-bundle": "^2.6 || ^3.0",
         "sainsburys/guzzle-oauth2-plugin": "^3.0",
         "sulu/sulu": "^2.0",
-        "symfony/config": "^4.3",
-        "symfony/http-kernel": "^4.3",
-        "symfony/messenger": "^4.3",
-        "symfony/templating": "^4.3",
-        "webmozart/assert": "^1.3"
+        "symfony/config": "^4.4 || ^5.0",
+        "symfony/dependency-injection": "^4.4 || ^5.0",
+        "symfony/framework-bundle": "^4.4 || ^5.0",
+        "symfony/http-foundation": "^4.4 || ^5.0",
+        "symfony/http-kernel": "^4.4 || ^5.0",
+        "symfony/messenger": "^4.4 || ^5.0"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.15",
+        "handcraftedinthealps/zendsearch": "^2.0",
         "jackalope/jackalope-doctrine-dbal": "^1.3.4",
-        "jangregor/phpstan-prophecy": "^0.3",
-        "matthiasnoback/symfony-dependency-injection-test": "^3.1",
-        "php-ffmpeg/php-ffmpeg": "^0.14.0",
-        "phpstan/phpstan": "^0.11",
-        "phpstan/phpstan-doctrine": "^0.11",
-        "phpstan/phpstan-symfony": "^0.11",
-        "phpstan/phpstan-webmozart-assert": "0.11.3",
+        "jangregor/phpstan-prophecy": "^0.8",
+        "matthiasnoback/symfony-dependency-injection-test": "^4.2",
+        "php-ffmpeg/php-ffmpeg": "^0.14",
+        "phpstan/phpstan": "^0.12",
+        "phpstan/phpstan-doctrine": "^0.12",
+        "phpstan/phpstan-symfony": "^0.12",
+        "phpstan/phpstan-webmozart-assert": "0.12",
         "phpunit/phpunit": "^8.2",
-        "symfony/browser-kit": "^4.3",
-        "symfony/dotenv": "^4.3",
-        "symfony/monolog-bundle": "^3.3",
-        "zendframework/zend-stdlib": "^2.3",
-        "zendframework/zendsearch": "@dev"
+        "symfony/browser-kit": "^4.4 || ^5.0",
+        "symfony/debug": "^4.4 || ^5.0",
```

Comment:
this is deprecated and there is not `5.0` version. but i dont think we are using it anywhere. so i guess we can just remove it?

Reply:
done this by myself 🙂